magda-registry-api/src/main/scala/au/csiro/data61/magda/registry/OrderByDef.scala

```diff
@@ -15,7 +19,7 @@ case class OrderByDef(field: String, dir: SQLSyntax = SQLSyntax.desc) {
       jsonPathItems: List[String] = Nil
   ) = {
     val columnRef = SQLSyntax.join(
-      (tableName.toList ++ List(columnName)).map(SqlHelper.escapeIdentifier(_)),
+      (tableName.toList ++ List(columnName)).map(SQLSyntax.createUnsafely(_)),
```

Comment:
We're still passing a `String` into `createUnsafely` - can we either use a type to restrict the columns/tables to known values, or check it at runtime?

Reply:
Updated~ It's now checking string pattern now~